static/app/views/releases/list/index.tsx

```diff
@@ -129,9 +135,48 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
        * uses shouldReload=true and there is no reloading happening.
        */
       forceCheck();
+      if (this.hasV2ReleaseUIEnabled) {
+        // Refetch new threshold statuses if  new releases are fetched
+        this.fetchThresholdStatuses();
+      }
     }
   }
 
+  fetchThresholdStatuses() {
+    const {selection, organization, api} = this.props;
+    const {releases} = this.state;
+
+    // Grab earliest release and latest release - then fetch all statuses within
+    let start = releases[0].dateCreated;
+    let end = releases[0].dateCreated;
```

Comment:
should this be `releases[releases.length-1]`?

Reply:
Nope - this is just setting a starting point for "end" - then we're iterating through all the releases and grabbing the latest end, and the earliest start

Reply:
oh right because release dates can overlap with multiple projects -- got it